custom/conf/app.example.ini

```diff
@@ -175,6 +175,37 @@ RUN_MODE = ; prod
 ;OFFLINE_MODE = false
 ;DISABLE_ROUTER_LOG = false
 ;;
+;; TLS Settings: Either ACME or manual
+;; (Other common TLS configuration are found before)
+;ENABLE_ACME = false
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; ACME automatic TLS settings
+;;
+;; ACME directory URL (e.g. LetsEncrypt's staging/testing URL: https://acme-staging-v02.api.letsencrypt.org/directory)
+;; Leave empty to default to LetsEncrypt's (production) URL
+;ACME_URL =
+;;
+;; Explicitly accept the ACME's TOS. The specific TOS cannot be retrieved at the moment.
+;; TODO: waiting on caddyserver/certmagic#165, after which this option should only be used if interactive setup is unavailable
```

Comment:
I don't think we should have TODOs in the config. Can you move these comments to code? If I understand correctly this option currently does nothing, so we could as well remove it.

Reply:
Noted. As of this commit, it is required to set to true in order for it to run. But it doesn't tell you the TOS, so I would prefer if the PR goes through. Although everything is in place if we get impatient and want to forego it for now

Reply:
At least remove the `TODO` marker. Keep in minde end users will read this config file, not developers.